docs/modules/ROOT/pages/overview.adoc

```diff
@@ -1,14 +1,104 @@
 = Hazelcast Command-Line Client (CLC)
-:description: Short description with main points
+:url-github-clc: https://github.com/hazelcast/hazelcast-cloud-cli/blob/master/README.md 
+:description: You can use the Hazelcast Command Line Client (CLC) to connect to and interact with clusters on Hazelcast Platform and {hazelcast-cloud}, direct from the command line or through scripts.
 
-What does the product do?
-what are some examples of where you’d use it?
-Who is it for?
-What problem does the product solve?
-What modes are available and what are they?
-What (if any) limitations does the product have?
-How do you get started?
+{description}
+
+The Hazelcast CLC simplifies the client setup process for Unix-like local environments. Within minutes of installation, you can start to perform common tasks on clusters.
+
+== Use Cases
+
+Example use cases for the Hazelcast CLC.
+
+=== Basic Cluster Management for Hazelcast Platform
+
+Perform xref:hzc-cluster.adoc[administration tasks] on multiple clusters without writing code. Instead, use <<interactive-mode, interactive mode>> to explore available commands.
+
+Check or change the state of a cluster, update a configuration option, then switch to another cluster, all from the command line. For maximum flexibility, you can set cluster connection details at the command line or using a xref:configuration.adoc[custom configuration file]. 
+
+NOTE: {hazelcast-cloud} has a separate link:{url-github-clc}[CLI for cluster management].
```

Comment:
I can not see the image on preview deployment. Maybe it is expected but just wanted to point out
https://deploy-preview-95--eclectic-sawine-19fcf1.netlify.app/docs/clc/0.1/overview.html